includes/post-hooks.php

```
@@ -621,6 +532,89 @@ function restore_and_sync_story( int $post_id, string $story_id, string $title )
+			'PostHooks',
+			'Failed to update story in Babbel',
+			array(
+				'post_id'  => $post_id,
+				'story_id' => $story_id,
+				'context'  => $log_context,
+				'fields'   => array_keys( $update_data ),
+				'error'    => $result['message'],
+			)
+		);
+	}
+}
```

PHPMD (2.15.0)

[error] The parameter $post_id is not named in camelCase. (undefined)

(CamelCaseParameterName)